LiveSPICE/LiveSPICE.csproj

```diff
@@ -129,7 +129,7 @@
     <Compile Include="Controls\Scope\SignalDisplay.cs" />
     <Compile Include="Utils\NullStream.cs" />
     <Compile Include="Utils\ObjectNotNullConverter.cs" />
-    <Compile Include="Utils\RedundantTaskScheduler.cs" />
+    <Compile Include="Utils\.cs" />
```

Comment:
This was intentional - I re-used your method for making simulation updates asynchronous, so I moved RedundantTaskScheduler from from the LiveSPICE project to the Util project so I wouldn't need a dependency on the LiveSPICE dll.

Reply:
Right, it's fine to move it to Util, my comment is about how the file in its old location is renamed from "RedundantTaskScheduler.cs" to ".cs" rather than deleted :) Seems likely to be some kind of VS glitch.

Reply:
Got it - fixed.